site/docs/plugins/session.md

```diff
@@ -694,6 +694,24 @@ The session plugin is able to enhance any storage adapter by adding more feature
 
 They can be installed using the `enhanceStorage` function.
 
+<CodeGroup>
+<CodeGroupItem title="TypeScript" active>
+
+```ts
+// Create a storage adapter.
+const storage = freeStorage<Enhance<SessionData>>(bot.token); // adjust this
```

Comment:
at the same time this is wrong. you don't need Enhance here

Reply:
that was the only way I can get it to work properly.

Reply:
or do you?..